src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -3,30 +3,78 @@
 import core.basesyntax.dao.CommentDao;
 import core.basesyntax.model.Comment;
 import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
 
 public class CommentDaoImpl extends AbstractDao implements CommentDao {
     public CommentDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
-    public Comment create(Comment entity) {
-        return null;
+    public Comment create(Comment comment) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(comment);
+            transaction.commit();
+            return comment;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert Content entity", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            session.beginTransaction();
+            Comment comment = session.get(Comment.class, id);
+            session.flush();
+            return comment;
```

Comment:
You can simplify it by returning get method directly

Reply:
i don't understand what you mean